### ISSUE TITLE
Rating Values (Assignment 0)

### ISSUE THREAD

**Issue by AndreaPerera** [CISC-873/Information#4]
Hello! 

I have a question about the rating values on Assignment 0. I noticed that the values are from 1-6. Is a value of 1 assumed to be that a product has 0 stars, and then likewise a value of 6 assumed to be 5 stars? 

Thanks ☺️

**Comment by steven-hh-ding** [comment]
@WeihanO  weihan, could you help double check? I think it is 1-5 rating but on Kaggle it shows bin size up to 6 (boundary?)

**Comment by AndreaPerera** [comment]
@steven-hh-ding I was also wondering what the difference between the "price" and "retail price" feature was. Thanks!

**Comment by theriaultr** [comment]
Hey Andrea, I see the ratings as ranging from 1-6 as well. 
@steven-hh-ding Does this matter for our Kaggle submission or should we change it to 0-5?
Thank you for your help!

**Comment by steven-hh-ding** [comment]
@theriaultr  yes weihan is checking and fixing that!! thanks for the heads-up!!

**Comment by WeihanO** [comment]
Hi guys. For the ratings: as said in our description, the dataset is not clean and it contains noise. So if any sample that does not conform to our description/requirement, please regard them as noise. You can filter them out or apply other denoising methods if you like. Thanks. And sorry for the confusion.

**Comment by WeihanO** [comment]
> @steven-hh-ding I was also wondering what the difference between the "price" and "retail price" feature was. Thanks!

The difference between price and retail_price is that:
- retail_price is the original price of products;
- price is the number when this dataset was generated. There can be fluctuations in price during the time (e.g. discounts/price increase). 
The attached picture shows the price/retail_price on a discounted product
![discounted_product](https://user-images.githubusercontent.com/60517240/94961417-9d0d6780-04c2-11eb-894b-e75821ad9b1e.png)
